src/autolabel/confidence.py

```diff
@@ -23,7 +23,7 @@
 
 
 class ConfidenceCalculator:
-    TTL_MS = 60 * 60 * 24 * 365 * 3 * 1000  # 3 years
```

Comment:
this is also used for the local sqlite db. maybe set it in refuel cloud only if possible

Reply:
The local cache might end up taking a lot of space over 3 years as well, right?